packages/vm/scripts/diffTester.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
For me this works, but I guess there is often some demand (and generall better) to define this as a general shell script and not a bash script, see e.g. https://stackoverflow.com/questions/5725296/difference-between-sh-and-bash ?

Reply:
I'll verify this works (i.e. just using `#!/bin/sh` and update if so).  I don't think there's anything fancy in this script but will confirm.